assets/app/data/crowbar/services/crowbar.factory.js

```diff
@@ -0,0 +1,84 @@
+(function() {
+
+    angular
+        .module('suseData.crowbar')
+        .factory('crowbarFactory', crowbarFactory);
+
+    crowbarFactory.$inject = ['$q', '$http', 'COMMON_API_V2_HEADERS'];
+    /* @ngInject */
+    function crowbarFactory($q, $http, COMMON_API_V2_HEADERS) {
+        var factory = {
+            getEntity: getEntity,
+            getRepositoriesChecks: getRepositoriesChecks,
+            upgrade: upgrade,
+            getUpgradeStatus: getUpgradeStatus
+        };
+
+        return factory;
+
+        /**
+         * Get Crowbar Entity
+         * 
+         * @return {Promise}
+         */
+        function getEntity() {
+
+            var requestOptions = {
+                method: 'GET',
+                url: '/api/crowbar',
+                headers: COMMON_API_V2_HEADERS
```

Comment:
It would simplify things a bit. We could use "decoupled service" (https://github.com/mgonto/restangular#decoupled-restangular-service) for the /api/ prefix and pre-configure it with required headers and other options if needed like I did here: https://github.com/skazi0/yaia/blob/master/app/static/js/invoices.js#L22-L27